src/components/TextareaField/TextareaField.stories.tsx

```diff
@@ -15,7 +15,7 @@ export default {
     label: 'Textarea Field',
     rows: 5,
     fieldNote: 'Longer Field description',
-    spellcheck: false,
```

Comment:
surprised this wasn't a typescript error before. Would is have been if you changed `as Meta<Args>` to `satisfies Meta<Args>`?

Reply:
it's possible. I can test that out. It's also possible typescript might allow both, and only warns at runtime... which react does for some kinds of things (the lower-case prop actually does what it says it should)